<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>서버 모니터링</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Server Monitoring</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RAM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DISK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>461</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 249, 195);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RAM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DISK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>lcdNumber</zorder>
    <zorder>lcdNumber_2</zorder>
    <zorder>lcdNumber_3</zorder>
    <zorder>lcdNumber_4</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP/PORT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ip">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_port">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>소켓</string>
    </property>
    <addaction name="actionSocket"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>데이터베이스</string>
    </property>
    <addaction name="actionDatabase"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSocket">
   <property name="text">
    <string>Socket</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
